Mixpanel/Mixpanel.m

```diff
@@ -1667,16 +1667,17 @@ - (void)notificationController:(MPNotificationViewController *)controller wasDis
     };
 
     if (status && controller.notification.callToActionURL) {
-        MixpanelDebug(@"%@ opening URL %@", self, controller.notification.callToActionURL);
-        BOOL success = [[UIApplication sharedApplication] openURL:controller.notification.callToActionURL];
+        [controller hideWithAnimation:NO completion:^{
```

Comment:
I apology for the delayed response.

> My only concern with this is that if a developer adds a custom URL to their notification they may not be aware that we are using this method internally and if they haven't registered the custom URL in the Info.plist then this code change will cause it to silently fail.

This would be the case if we start using `canOpenURL:` before opening the URL. But we don't do that here. We still use same `openURL:` which seem to have the same behavior.

I think it's correct here to use `openURL:` as before. The only thing I didn't have a strong opinion on is whether to dismiss notification view controller with animation or without. We have shipped this code in our app a few month ago and didn't have any problems with it. I think it's ok to merge without animation and if somebody will bring a question, it can be changed then.